config/zones.json

```diff
@@ -5848,12 +5850,15 @@
       ]
     ],
     "capacity": {
-      "biomass": 151,
-      "hydro": 4810,
-      "hydro storage": 1487,
+      "biomass": 230,
+      "coal": 17242,
+      "gas": 4600
```

Comment:
, missing

Reply:
I was just about to look out for them but thank you, makes it a lot quicker for me.